shared/src/bundles/user-details/validation-schemas/user-details-update.validation-schema.ts

```diff
@@ -68,6 +68,8 @@ const userDetailsUpdate = joi
         companyWebsite: joi.string().trim().uri(),
         employerPosition: joi.string().trim(),
         cvId: joi.string().trim(),
+        talentBadges: joi.array().items(joi.string().trim()),
```

Comment:
same

Reply:
as I understood @MaksymGavatuk about talent badges, we should recieve from frontend only array of strings = Ids

Reply:
Yes it just string[]